### DISCUSSION TITLE
Building Frigate

### DISCUSSION THREAD

**Discussion by YBonline** [blakeblackshear/frigate#6299] [Ask A Question] [answered]
I wanted to run the dev version of Frigate as I added a few more cameras and I'm running into issue #5818 pretty badly as I reach 40 cameras. I see this is fixed by PR #6248 so I wanted to test that and start using it so I can add additional cameras...

However, when I run make it builds a container that runs a fake Frigate process according to the logs. I'd like to try to keep it in a docker container, so I tried following the instructions for making an official build in the contributing section, and have been having a number of errors. Is there a recommended way to do this or am I doing something wrong? Also, I know its super early, but when do you expect there to be a first alpha of 0.13 so I can decide if I should wait or continue to try to work through my build issues?

**Comment by NickM-27** [comment]
Dev builds are made on each commit to dev branch, I'd suggest just running those.

**Reply by YBonline** [reply]
How do you access those?

**Reply by NickM-27** [reply, marked as answer]
https://github.com/blakeblackshear/frigate/pkgs/container/frigate
